src/lib/controllers/Client/index.js

```diff
@@ -0,0 +1,14 @@
+class ClientController {
+  static disconnectionHandler() {
+    console.log('Client close');
+    // TODO
+  }
+
+  static connectHandler(socket, moduleContainer) {
+    console.log('Client open');
+    socket.on('bridge-data', moduleContainer.handle);
+    socket.on('disconnect', this.disconnectionHandler);
+  }
+}
+
+module.exports = ClientController;
```

Comment:
Why to name it `ClientController` ?

Reply:
Because i only use it for websocket client.
Connect give a socket connected to one client so I thought it was better